### ISSUE TITLE
Causes plasmashell to crash.

### ISSUE THREAD

**Issue by akwala** [Davide-sd/ip_address#10]
v. 1.2.5 on Kubuntu 18.04

```
kernel: plasmashell[26977]: segfault at 8 ip 00007fc317a097fa sp 00007fff495a18d0 error 4 in libQt5Location.so.5.9.5[7fc317995000+1a5000]
```

The segfault is preceded by repeating errors for several NetworkManager actions -- see attached file.
[PublicIP-plasmashell-NM.txt](https://github.com/Davide-sd/ip_address/files/4685291/PublicIP-plasmashell-NM.txt)

**Comment by Davide-sd** [comment]
Hello @akwala ,

Please provide further infos: Os, version, KDE version... 

From the errors I assume you have enabled the option "display VPN icon status". Open the terminal and run the following command, copy the result here! 

`nmcli c show --active | grep -E "vpn|tun"`

In the meantime, disable the option "display VPN icon status".

**Comment by akwala** [comment]
> Please provide further infos: Os, version, KDE version...

Kubuntu 18.04
![image](https://user-images.githubusercontent.com/1786/82961407-4bd8ee80-9f8b-11ea-9814-ac6a8b2399c5.png)

> From the errors I assume you have enabled the option "display VPN icon status". 

With the VPN status display enabled, the crash happened soon after desktop login.
With the VPN status display disabled, the crash happened when I clicked the widget.

> Open the terminal and run the following command, copy the result here!
> `nmcli c show --active | grep -E "vpn|tun"`

Output is of the form:

```
<connection_ID>  <UUID>  vpn     wlp0s20f3 
tun0             <UUID>  tun     tun0
```

**Comment by Davide-sd** [comment]
Were you using the previous version of this widget, the one without the VPN status icon? If so, did you encountered any errors before?

**Comment by akwala** [comment]
> Were you using the previous version of this widget, the one without the VPN status icon? If so, did you encountered any errors before?

Yes, I've been using the widget for quite some time. I never encountered these errors until v. 1.2.5 which I installed yesterday.

> Also, please run the command `sudo apt-get install libqt5positioning5 libqt5location5 qtlocation5-dev qtpositioning5-dev qml-module-qtlocation qml-module-qtpositioning` and past the entire output.
```
Reading package lists... Done
Building dependency tree       
Reading state information... Done
libqt5location5 is already the newest version (5.9.5+dfsg-0ubuntu2).
libqt5positioning5 is already the newest version (5.9.5+dfsg-0ubuntu2).
qml-module-qtlocation is already the newest version (5.9.5+dfsg-0ubuntu2).
qml-module-qtpositioning is already the newest version (5.9.5+dfsg-0ubuntu2).
qtlocation5-dev is already the newest version (5.9.5+dfsg-0ubuntu2).
qtpositioning5-dev is already the newest version (5.9.5+dfsg-0ubuntu2).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
```

**Comment by akwala** [comment]
Because it kills plasmashell, this issue makes it impossible to launch a desktop session without first manually removing the widget using a text console.

Below are my config settings with the IP and VPN status display disabled. With these, the crash occurs a couple of seconds after I click the widget.
![Screenshot_20200527_120020](https://user-images.githubusercontent.com/1786/83047393-542a3b80-a016-11ea-86e4-de9cb106635b.png)
![Screenshot_20200527_120045](https://user-images.githubusercontent.com/1786/83047399-555b6880-a016-11ea-8f13-4081469e37db.png)

**Comment by Davide-sd** [comment]
This is bugging me, for real. 

In the meantime, remove the current widget, uninstall it. I've uploaded the previous version to pling, (the file is called `v1.1.com.github.davide-sd.ipaddress.pasmoid`): download it and install it.

Back to the problem: it may be caused by some library. You are using a LTS version of Ubuntu, maybe there were some library update that causes this sort of crash. It's really hard to say.

If I provide the necessary instructions to try to debug this problem from your pc, would you please do that and report the findings?

**Comment by Davide-sd** [comment]
Ahhh man, now I understand: it is most likely the option "Show host name" to cause the problem. It just happened to me too. If you are able to deactivate it, maybe it's going to work... Let me know.

Need to understand why that happens..

**Comment by akwala** [comment]
> Ahhh man, now I understand: it is most likely the option "Show host name" to cause the problem. It just happened to me too. If you are able to deactivate it, maybe it's going to work... Let me know.

Disabled 'Show host name'. Same behavior.

**Comment by akwala** [comment]
Happy to report that I haven't seen this issue in Kubuntu 20.04 since I upgraded a couple of days ago.
🎉

Operating System: Kubuntu 20.04
KDE Plasma Version: 5.18.5
KDE Frameworks Version: 5.68.0
Qt Version: 5.12.8
Kernel Version: 5.4.0-52-generic
OS Type: 64-bit
